iconservice/prep/engine.py

```diff
@@ -586,6 +609,24 @@ def handle_set_prep(cls, context: 'IconScoreContext', params: dict):
             del kwargs[ConstantKeys.P2P_ENDPOINT]
             kwargs["p2p_endpoint"] = p2p_endpoint
 
+        if ConstantKeys.NODE_ADDRESS in kwargs:
+            node_address: 'Address' = kwargs[ConstantKeys.NODE_ADDRESS]
+
+            if context.revision < Revision.DIVIDE_NODE_ADDRESS.value:
+                if dirty_prep.address != node_address:
+                    raise InvalidParamsException(f"Mismatch nodeAddress and address. "
+                                                 f"'divide node address' revision need to be accepted"
+                                                 f"(revision: {Revision.DIVIDE_NODE_ADDRESS.value})")
+
+            self._validate_node_address(context, node_address)
+
+            del kwargs[ConstantKeys.NODE_ADDRESS]
+            kwargs["node_address"] = node_address
+            if node_address != address:
+                context.prep_address_converter.add_prev_node_address_mapper(key=dirty_prep.node_address, value=address)
+                context.prep_address_converter.delete_node_address_mapper(dirty_prep.node_address)
+                context.prep_address_converter.add_node_address_mapper(key=node_address, value=address)
```

Comment:
I think that it is possible to optimize this code block further.
Why are prev_node_address-related methods needed?
And how about removing "_mapper" from method names?